src/librustc/ty/print/pretty.rs

```diff
@@ -922,10 +922,18 @@ pub trait PrettyPrinter<'tcx>:
                         .get_bytes(&self.tcx(), ptr, Size::from_bytes(n)).unwrap())
                 },
                 (ConstValue::Slice { data, start, end }, ty::Slice(t)) if *t == u8 => {
-                    Some(&data.alloc.bytes[start..end])
+                    match data.get_raw_bytes(start..end) {
+                        BytesOrElidedUndef::MaybeBytes(bytes) => Some(bytes),
+                        // FIXME: I don't know how to handle this. Probably a bug.
+                        BytesOrElidedUndef::ElidedUndef => unimplemented!(),
```

Comment:
yea, just use `bug!` here

Reply:
although using `None` doesn't hurt for pretty printing, so that would be ok, too